tests/src/tracing/runtimeeventsource/RuntimeEventSourceTest.cs

```diff
@@ -0,0 +1,68 @@
+using System;
+using System.IO;
+using System.Diagnostics.Tracing;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Tracing.Tests.Common;
+
+namespace Tracing.Tests
+{
+    public sealed class RuntimeEventSourceTest
+    {
+        static int Main(string[] args)
+        {
+            // Get the RuntimeEventSource.
+            EventSource eventSource = RuntimeEventSource.Log;
+
+            // Create of EventListener.
+            using (SimpleEventListener listener = new SimpleEventListener())
+            {
+                // Trigger the allocator task.
+                System.Threading.Tasks.Task.Run(new Action(Allocator));
+
+                // Enable events.
+                listener.EnableEvents(eventSource, EventLevel.Verbose, (EventKeywords)(-1));
+
+                // Wait for events.
+                Thread.Sleep(5000);
+
+                GC.Collect(2, GCCollectionMode.Forced);
+
+                // Wait for more events.
+                Thread.Sleep(5000);
```

Comment:
Missing `Assert.True(listener.EventCount > 0);`

Reply:
Thanks.  I have been using this as a local test case to catch exceptions, asserts, etc.  I've added this, but will also be adding more test cases in upcoming PRs.